src/page/Error/ErrorPage.tsx

```
@@ -1,45 +1,77 @@
-import { useNavigate } from 'react-router-dom';
 import { styled } from 'styled-components';
+import { useNavigate } from 'react-router-dom';
+import { ReactComponent as Error } from '@/assets/icons/Error.svg';
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)